CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/InfoMapPanel.java

```diff
@@ -63,9 +67,8 @@ public Config getConfig() {
     }
 
     /**
-     * This method is called from within the constructor to initialize the form.
-     * WARNING: Do NOT modify this code. The content of this method is always
-     * regenerated by the Form Editor.
+     * This method is called from within the constructor to initialize the form. WARNING: Do NOT modify this code. The
```

Comment:
I assume there was some auto-formatting here or similar just looking at the rest of the PR but I would move the warning part to a new line